### PR TITLE
Modernize Python 2 code to get ready for Python 3

### LINKED ISSUE #182: Undefined names 'WHITE' and 'PALETTE' in utils/color.py
$ __python2 -m flake8 . --count --select=E9,F821,F822,F823 --show-source --statistics__
```
./utils/color.py:17:22: F821 undefined name 'WHITE'
        return Color(WHITE)
                     ^
./utils/color.py:67:26: F821 undefined name 'PALETTE'
    return random.choice(PALETTE)                         ^
2     F821 undefined name 'WHITE'
```

### PR DESCRIPTION
**Pull request by cclauss**
Some minimal, "safe" changes to make the code syntax compatible with both Python 2 and Python 3.
* Run __futurize --stage1__ http://python-future.org/futurize_cheatsheet.html
* Adjust adjust some __lambda__ and __tuple__ expressions to be compatible with Python 3.

Fixes #182 and helps with  #128

__Ready for review.__  Python 3 syntax compatibility does _not_ mean that the port is complete.

### CODE REVIEW

File: active_projects/eop/bayes.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def indicate_straight(self):
             card.generate_target()
 
         straight_cards = VGroup(*it.chain(
-            you.hand.target,
```
[the remark this answers is not in the document]
Comment (cclauss) [reply active_projects/eop/bayes.py]: Running __autopep8__ or __yapf__ on the codebase would be the best approach in my book.  If you do that then I could rebase once that change is merged.

[the remark this answers is not in the document]
Comment (vors) [reply active_projects/eop/bayes.py]: I'm in the same shoes as you: no write access :)

[the remark this answers is not in the document]
Comment (cclauss) [reply active_projects/eop/bayes.py]: Maybe also run __isort__ across the codebase to organize the imports.

[the remark this answers is not in the document]
Comment (cclauss) [reply active_projects/eop/bayes.py]: It could be done as a PR.

[the remark this answers is not in the document]
Comment (cclauss) [reply active_projects/eop/bayes.py]: Too many conflicts... closing for now.

**Pull request closed without merging**